salesagility/SuiteCRM
# Fixed alerts not counting down

## Description







These fixes make the alerts count down so that the user gets the notification reminder.

relates to issue #2082 
## Motivation and Context



When a user set up a reminder on calls or meetings they do not receive a desktop notification.
## How To Test This



Create a popup reminder in calls and meetings.
Wait for the notification.

If you want to check the counting down you can run the following in the browser console:

> setInterval(function() { console.log(alertList[0].time) }, 1000);
## Types of changes


### Final checklist
